lib/virt_autotest/utils.pm

```diff
@@ -105,10 +105,23 @@ sub is_xen_host {
     return get_var("XEN") || check_var("SYSTEM_ROLE", "xen") || check_var("HOST_HYPERVISOR", "xen") || check_var("REGRESSION", "xen-hypervisor");
 }
 
+#check kernels by grep keywords from journals
+#welcome everybody to extend this function
+sub check_kernel {
```

Comment:
Please give a more specific name to your function. IMHO. check_host, check_guest, check_kernel are not the meaningful names.

Reply:
yes, have the same idea. according to function name of check_kernel (), sound like to check with loaded kernel module on SLES VM host. but,  it need to figure out if there is kernel core dump or kernel call trace from  journalct (i.e system_call or system_log)  for named function check_kernel ().  named as check_kernel_core_dump() or check_kernel_call_trace() that make much more sense as the suggestion.